cdk/recording_events_table.py

```diff
@@ -0,0 +1,33 @@
+from aws_cdk import Stack, RemovalPolicy, aws_dynamodb as dynamodb
+from constructs import Construct
+from . import names
+
+
+class RecordingEvents(Construct):
+    def __init__(self, scope: Construct, id: str):
+        """
+        This table records the times of some webhook events.
+        """
+        super().__init__(scope, id)
+        stack_name = Stack.of(self).stack_name
+
+        self.table = dynamodb.Table(
+            self,
+            "table",
+            table_name=f"{stack_name}-{names.RECORDING_EVENTS_TABLE}",
+            partition_key=dynamodb.Attribute(
+                name="zoom_uuid",
+                type=dynamodb.AttributeType.STRING,
+            ),
+            billing_mode=dynamodb.BillingMode.PAY_PER_REQUEST,
```

Comment:
oh i guess if we used `PROVISIONED` it could get used up and that would be bad?

Reply:
I didn't look up; I copied from the status table because it's similar.

Reply:
I think on-demand is best because it's unpredictable traffic and we want to pay only for what we use (https://docs.aws.amazon.com/wellarchitected/latest/serverless-applications-lens/capacity.html)